src/components/Results/Sort/Sort.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from "react";
+import { ISortProps } from "./Sort.types";
+import "./Sort.scss";
+
+export const Sort: FC<ISortProps> = ({
+  options,
+  onOptionChange,
+  sortLabel,
+}) => {
+  return (
+    <div className="sort">
+      <label className="sort__label" htmlFor="sort">
```

Comment:
In latest commits I decided to go with classnames rather than bem-react/classnames because last option requires more adjustments to adapt to requirements